package.json

```diff
@@ -94,8 +95,8 @@
     "to-boolean": "^1.0.0",
     "ts-node": "^8.1.0",
     "tsconfig-paths": "^3.8.0",
-    "typescript": "^3.4.5",
     "tslint-microsoft-contrib": "^6.1.1",
+    "typescript": "^2.9.2",
```

Comment:
why was this changed? it's been downgraded?

I assume this is from merging in master?

Reply:
I think it happened because I did some local npm installation... Now upgraded to match master and ithc